settings.gradle

```diff
@@ -146,13 +146,11 @@ if (isCiServer || isAirbyteCI) {
             enabled = isAirbyteCI
         }
         remote(com.github.burrunan.s3cache.AwsS3BuildCache) {
-            region = 'us-east-2'
-            bucket = 'airbyte-buildcache'
-            prefix = 'cache/'
-            push = isCiServer
-            enabled = isCiServer && !isAirbyteCI
-            // Credentials will be taken from  S3_BUILD_CACHE_... environment variables
-            // anonymous access will be used if environment variables are missing
+            region = 'us-west-2' // close to dagger runners
+            bucket = 'ab-ci-cache'
+            prefix = "${System.getProperty('s3BuildCachePrefix', 'connectors')}-ci-cache/"
```

Comment:
`s3BuildCachePrefix` is this and env var? 
As this repo is for connectors can we just make `prefix` static with a value of `connectors-ci-cache`?

Reply:
No it's an optional gradle property (see gradle.py) and in airbyte-ci it is set deliberately to isolate the build cache on a per-connector basis. Among other things this will make the build times which we measure for source-postgres less chaotic.

Reply:
Ok got it! Thanks!